### ISSUE TITLE
Ads don't get blocked on Youtube tabs that Firefox restores

### ISSUE THREAD

**Issue by sim642** [gorhill/uBlock#3270]
### Describe the issue
When Firefox crashes and is restarted, it offers to restore the previous tabs. When a restored tab is a Youtube video tab, the tab loads and all the ads are not blocked. Refreshing the page or visiting any Youtube videos at any later time properly has them blocked.

### One or more specific URLs where the issue occurs
https://www.youtube.com/watch?v=ehV_I7LEXbA

### Steps for anyone to reproduce the issue
1. Have Firefox open with active Youtube video tab
2. Crash or kill Firefox
3. Restart Firefox and restore the tabs
4. Observe that the Youtube video tab which opens has ads which are normally blocked (e.g. the banner on above the right hand side suggested videos list)
5. Refresh the page to see that they actually get blocked properly if the page is loaded not as part of tab restore

### Your settings
- OS/version: Xubuntu 16.04
- Browser/version: Firefox 56.0
- uBlock Origin version: uBlock Origin 1.14.18

##### Your filter lists
Default filter lists + EST

**Comment by gorhill** [comment]
Duplicate of #1327.
